packages/cw1155/src/event.rs

```diff
@@ -21,13 +20,19 @@ impl TransferEvent {
 
 impl From<TransferEvent> for Event {
     fn from(event: TransferEvent) -> Self {
-        Event::new("transfer_tokens").add_attributes(
-            vec![
-                attr("sender", event.sender.as_str()),
-                attr("recipient", event.recipient.as_str()),
-                attr("tokens", event.tokens.iter().map(|t| t.to_string()).collect::<Vec<_>>().join(",")),
-            ]
-        )
+        Event::new("transfer_tokens").add_attributes(vec![
```

Comment:
@apolloshab should we name this `transfer_nfts` to be more consistent with the current CW721 standard?

Reply:
agreed, thinking of changing to *_single and *_batch for everything